frontend/hooks/player.ts

```diff
@@ -10,21 +10,38 @@ import {
 
 export function usePlayer(
     audio: string
-): [boolean, Dispatch<SetStateAction<Boolean>>, MutableRefObject<() => void>] {
+): [
+    boolean,
+    Dispatch<SetStateAction<Boolean>>,
+    MutableRefObject<(string) => void>
```

Comment:
Seeing as this type is commonly passed as props, consider adding a type alias and use it so we can easily change the type later on if needed?